mllib/src/main/scala/org/apache/spark/ml/r/SparkRWrappers.scala

```diff
@@ -52,6 +53,43 @@ private[r] object SparkRWrappers {
     pipeline.fit(df)
   }
 
+  def fitAFTSurvivalRegression(
+      value: String,
+      df: DataFrame): PipelineModel = {
+
+    def formulaRewrite(value: String): (String, String) = {
+      var rewrited: String = null
+      var censorCol: String = null
+
+      val regex = "^Surv\\(([^,]*),([^,]*)\\)\\s*\\~\\s*(.*)".r
+      try {
+        val regex(label, censor, features) = value
+        // TODO: Support dot operator.
+        if (features.contains(".")) {
+          throw new UnsupportedOperationException(
+            "Terms of survreg formula can not support dot operator.")
```

Comment:
We will not support dot operator currently, because dot will be expanded to all column names including the ```censorCol``` which is unexpected. 
If terms of survreg formula only include ```.```, we can expand it to all feature column names at here. But if the formula contains ```.:.``` or ```.:x```, we can not handle it here. So I choose to not support any kind of dot in term.